src/AutoRest.CSharp/Mgmt/AutoRest/MgmtTarget.cs

```diff
@@ -235,7 +236,17 @@ public static async Task ExecuteAsync(GeneratedCodeWorkspace project)
             //if (_overriddenProjectFilenames.TryGetValue(project, out var overriddenFilenames))
             //    throw new InvalidOperationException($"At least one file was overridden during the generation process. Filenames are: {string.Join(", ", overriddenFilenames)}");
 
-            var modelsToKeep = Configuration.MgmtConfiguration.KeepOrphanedModels.ToImmutableHashSet();
+            var emitContext = Configuration.MgmtConfiguration.IsArmCore ? Configuration.Namespace == "Azure.ResourceManager" ? true : false : true;
```

Comment:
curious about the false case here.
Checked in azure-sdk-for-net repo, `arm-core` only occurs in 2 place: 
https://github.com/Azure/azure-sdk-for-net/blob/351f0401db53b6709159a04139cd41af796407df/sdk/resourcemanager/Azure.ResourceManager/src/autorest.md?plain=1#L7
https://github.com/Azure/azure-sdk-for-net/blob/351f0401db53b6709159a04139cd41af796407df/sdk/resourcemanager/Azure.ResourceManager/eng/autorest-resources.md?plain=1#L6
seems like arm-core always comes within `Azure.ResourceManager`

Reply:
arm-core also happens with Azure.ResourceManager.Resources and Azure.ResourceManager.ManagementGroup namespaces inside the Azure.ResourceManager library.  We do a batch generation of 3 libraries into one.